test/diff/operation/out.nix

```
@@ -273,4 +273,17 @@
         z = 30;
       }
   )
+
+  # Experimental pipe operators
+  (
+    a // b
+    |> f "very long argument should justify splitting this over multiple lines"
+    |> g { }
+  )
+
+  (
+    g { }
+    <| f "very long argument should justify splitting this over multiple lines"
+    <| a // b
+  )
```

What I like about the `|>` formatting is that the functions, that have an equal role, are formatted equally.
This makes it easy to read. 

That is not the case here. This formatting makes it seem as if `g { }` is special and `f "..."` and `a // b` are similar, while the opposite is true.

```suggestion
  (
    g { } <|
    f "very long argument should justify splitting this over multiple lines" <|
      a // b
  )
```

A similar effect could be achieved by indenting the first function, but that's a little unusual, and it doesn't make the function operands look quite as equal.

```suggestion
  (
       g { }
    <| f "very long argument should justify splitting this over multiple lines"
    <| a // b
  )
```

---

The first suggested style has the problem that lines might be arbitrarily long, which makes it unclear how the lines relate to, without looking to the right. This is why the standard [always puts operators in front](https://github.com/NixOS/nixfmt/blob/e819b2d0f9173f0c73d2e1fb4bc5155362046653/standard.md?plain=1#L765).

The second suggested style violates the [indentation rule](https://github.com/NixOS/nixfmt/blob/e819b2d0f9173f0c73d2e1fb4bc5155362046653/standard.md?plain=1#L232) of the standard.

But we also agree that it's not ideal for the first function to look different. For now we don't want to block on this and just go ahead with this PR as is.

It would also be possible for https://github.com/NixOS/rfcs/pull/148 to specify how it should be formatted (though it doesn't actually specify a `<|` operator right now).